pkg/Microsoft.Private.PackageBaseline/stable.packages.props

```diff
@@ -1356,5 +1356,49 @@
     <StablePackage Include="System.Runtime.Serialization.Xml">
       <Version>4.1.2</Version>
     </StablePackage>
+
+    <!-- Servicing 1.0.5 wave -->
+    <StablePackage Include="Microsoft.NETCore.Targets">
+      <Version>1.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Http">
+      <Version>4.1.2</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Http.WebHttpHandler">
+      <Version>4.0.2</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.Security">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="System.Net.WebSockets.Client">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="System.Text.Encodings.Web">
+      <Version>4.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.debian.8-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.2</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.fedora.23-x64.runtime.native.System.Security.Cryptography">
+      <Version>1.0.2</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.native.System.Security.Cryptography">
+      <Version>4.0.1</Version>
```

Comment:
The others are 1.0.2 and this one is 4.0.1. This is correct, I also did a double take. It has to do with the fact that we versioned the shim packages at 1.0.0 at first and in .NET Core 1.1 we moved them all to 4.3.0 as well.